ci_framework/roles/rhol_crc/defaults/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+# All variables within this role should have a prefix of "cifmw_rhol_crc"
+
+# Binary variables:
+rhol_crc_version: 2.14.0
+rhol_crc_tarball_url: https://mirror.openshift.com/pub/openshift-v4/clients/crc/{{ rhol_crc_version }}/crc-linux-amd64.tar.xz
+rhol_crc_binary_folder: "{{ lookup('env','HOME') }}/bin"
```

Comment:
I'm not sure about `default( ansible_user_dir ~ '/ci-framework')`. Do we want to keep the binary inside of the project folder?

I've modified like this.

In case  `cifmw_basedir` isn't defined it will take the base dir of the project based with `git rev-parse`. The `cd` doesn't affect the rest of the tasks. There's no way to give as an argument a directory to check it.

```
cifmw_rhol_crc_basedir: "{{ cifmw_basedir | default(lookup('ansible.builtin.pipe', 'cd {{ role_path }}; git rev-parse --show-toplevel')) }}"                           
```
Let me know what do you think.

Reply:
Oh. wait. ok. There's a confusion - and it's my fault. Should make something better for the data directory.
I don't expect people to get the `ci-framework` project/repository (so the git clone) in ~/ - especially since the project would be cloned from within `install_yamls` and, therefore, located somewhere in there (to be defined).

The ~/ci-framework is another directory, created by the setup role (and I think I have to rename it, else we'll face a real issue pretty soon).

So there's not need to make some `cd` nor anything like that. Maybe we can rename the ~/ci-framework (holding the data, artifacts and all) to something else, why not with some randomized suffix?

Reply:
I get it now. Changes done. Thx.